sympy/core/basic.py

```diff
@@ -1855,6 +1855,22 @@ def _exec_constructor_postprocessors(cls, obj):
 
         return obj
 
+    def _sage_(self):
```

Comment:
Just a heads up, there are a few SymPy classes that do not subclass from Basic. I don't know if you interface with them in Sage, but if you do, you shouldn't assume that they will have this logic.

Reply:
Thanks! Do you have an example/pointer so I can check this?

Reply:
The main example is Matrix, which defaults to a mutable version (so can't subclass from Basic). There are also some things in physics, which it is unlikely you are using. Also a lot of internal stuff like the classes in the polys.

Reply:
Thanks. I've taken care of `Matrix` in 554ca46cf4de32b9ffa4af81a667d38d6f220bde